src/interactive/dropdown.test.jsx

```diff
@@ -62,16 +57,9 @@ describe('Dropdown', () => {
 		/>
 	);
 	const wrapper = mount(dropdownJSX);
-	const portalWrapper = wrapper.find(Portal);
-
-	it('renders into DOM', () => {
-		expect(wrapper).toMatchSnapshot();
```

Comment:
Checking snapshots gave me an error after these updates:
```enzyme RangeError: Invalid string length```

I coudn't find a good way to fix it, so I'm testing everything else instead. I'd love some recommendations on how we can make sure we're getting good coverage